content/docs/0.6.0/reference/monitoring/prometheus/index.md

```diff
@@ -6,4 +6,93 @@ icon: setup
 keywords: setup
 ---
 
-No setup is needed for Prometheus as it is conducted in the tutorials [Onboarding a Service](../../../usecases/onboard-carts-service/) and [Self-healing with Keptn](../../../usecases/self-healing-with-keptn/).
+In order to evaluate the quality gates and allow self-healing in production, we have to set up monitoring to get the needed data.
+
+## Prerequisites
+
+- To setup Prometheus monitoring for you Kubernetes cluster, a Keptn project and at least one onboarded service must be available. For example, finish the [Onboarding a Service](../onboard-carts-service/) tutorial or onboard your own service.
+
+## Setup Prometheus
+
+After creating a project and service, you can setup Prometheus monitoring and configure scrape jobs using the Keptn CLI. 
+
+* Execute the following command to set up the rules for the *Prometheus Alerting Manager*:
+
+  ```
+  keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAME
+  ```
+
+* To verify that the Prometheus scrape jobs are correctly set up, you can access Prometheus by enabling port-forwarding for the prometheus-service:
+
+    ```console
+    kubectl port-forward svc/prometheus-service 8080 -n monitoring
+    ```
+  Prometheus is then available on [localhost:8080/targets](http://localhost:8080/targets) where you can see the targets for the service:
+
+    {{< popup_image link="./assets/prometheus-targets.png" caption="Prometheus Targets">}}
+
+* Finally, the **prometheus-service** need to be deployed to make Keptn aware of Prometheus. To deploy this service, please complete following tasks: 
+
+  ```console
+  git clone --branch 0.3.0 https://github.com/keptn-contrib/prometheus-service --single-branch
+  ```
+
+  ```console
+  cd prometheus-service/deploy/
+  ```
+
+  ```console
+  kubectl apply -f distributor.yaml
+  ```
+
+  ```console
+  kubectl apply -f service.yaml
+  ```
+  
+## Setup Prometheus SLI provider 
+
+During the evaluation of a quality gate, the Prometheus SLI provider is required that is implemented by an internal Keptn service, the *prometheus-sli-service*. This service will fetch the values for the SLIs that are referenced in a SLO configuration.
+
```

Comment:
I just noticed that you wrote this guide such that it is very similar to the Dynatrace SLI Provider guide, thank you for that!

I like this way, and I would only want to mention that as far as I know the **prometheus-sli-provider** is automatically installed. Therefore I would recommend to add this to the tutorial:

**Verify if the prometheus sli provider has already been installed**:
```console
kubectl get deployment -n keptn -o wide prometheus-sli-service
```

Reply:
Good idea. I added this step.